.browserslistrc

```diff
@@ -0,0 +1,17 @@
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
+# For additional information regarding the format and rule options, please see:
+# https://github.com/browserslist/browserslist#queries
```

Comment:
add a .gitignore file and put mode_modules in ignore.

Reply:
sure I will add